web/src/layouts/UsersAdministrationLayout/UsersAdministrationLayout.tsx

```diff
@@ -0,0 +1,112 @@
+import { FC, ReactNode } from "react";
+import { useLocation } from "react-router-dom";
+
+import { Box, Theme, Button as MuiButton } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+import { Link } from "@components/Link/Link";
+import { ArrowLeftIcon } from "@components/icons/ArrowLeftIcon/ArrowLeftIcon";
+import { PageLayout } from "@layouts/PageLayout/PageLayout";
+import { TopAppBar } from "@layouts/TopAppBar/TopAppBar";
+import { rgbaColors } from "@themes/palette";
+import { Paths } from "@routing/paths";
+
+interface Props {
+  onBackClick?: () => void;
+  buttons: ReactNode;
+}
+
+export const UsersAdministrationLayout: FC<Props> = ({
+  children,
+  onBackClick,
+  buttons,
+}) => {
+  const theme = useTheme<Theme>();
+  const location = useLocation();
+
+  const isUsersTab = location.pathname.split("/")[2] === "users";
```

Comment:
This should be turned into a generic method, so we'll be able to reuse it on other tabs in the future.